include/benchmark/benchmark.h

```diff
@@ -1732,6 +1738,19 @@ inline const char* GetTimeUnitString(TimeUnit unit) {
   BENCHMARK_UNREACHABLE();
 }
 
+inline TimeUnit GetTimeUnitFromString(std::string unit_string) {
```

Comment:
Why in `benchmark.h`? Only used by `benchmark.cc`

Reply:
Because the pair of this function is in the header but as I see that is used in other files as well.

Reply:
0847e780bf383a27269db0bcb8ca1449131a71c5